bot/code_coverage_bot/uploader.py

```diff
@@ -42,7 +42,7 @@ def gcp(repository, revision, report):
     logger.info('Uploaded {} on {}'.format(path, bucket))
 
     # Trigger ingestion on backend
-    retry(lambda: gcp_ingest(repository, revision), retries=5)
+    retry(lambda: gcp_ingest(repository, revision), retries=10, wait_between_retries=60)
```

Comment:
I wonder if 10 minutes is not a bit too much, maybe we should log some timing information to make sure we don't always wait too long